features/input-date-time.yml

```diff
@@ -0,0 +1,16 @@
+name: <input> date, datetime-local, time, and week
+description: The `<input>` element allows the user to input date and time information to a form.
```

Comment:
(non-blocking feedback)

Use of `allows` in all these descriptions seems ok per https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#allows.

However, the attribute value seems more like a *constraint* on the input the user is able to enter so it reads a bit ambiguously as opposed to "allows the user to *only* input date and time..." or "restricts input to..." or something like that.